src/sources/dnstap/mod.rs

```diff
@@ -30,8 +30,9 @@ use vector_core::config::{LegacyKey, LogNamespace};
 #[configurable_component(source("dnstap"))]
 #[derive(Clone, Debug)]
 pub struct DnstapConfig {
-    /// Maximum length, in bytes, that a frame can be.
+    /// Max dnstap frame length that the dnstap source can handle.
```

Comment:
```suggestion
    /// Max DNSTAP frame length that the dnstap source can handle.
```

Not sure, just based off the comments below.